documentation/how-to/gnss.en.md

```diff
@@ -267,6 +267,10 @@ Examples:
             is `NULL`. - `@position_source_name` > The value is
             `manual`.
 
+Info for GNSS Z value with Vertical grid shift in use:
+![image](https://github.com/opengisch/QField-docs/assets/23143342/2f0d6c78-7b38-47ab-bfd9-2e39d048cfa3)
```

Comment:
My five cents on this is if possible we store the images in our own repo.

Reply:
Cannot comment on the contents though, I guess @lucienicolier is a better reviewer.

Reply:
I moved the image to a our own repo